packages/web-components/src/components/masthead/left-nav-overlay.ts

```diff
@@ -31,7 +31,8 @@ class DDSLeftNavOverlay extends LitElement {
     if (changedProperties.has('active')) {
       const doc = this.getRootNode() as Document;
       if (this.active) {
-        const masthead: HTMLElement | null = doc?.querySelector('dds-masthead');
+        // TODO: remove when masthead can account for banners above
+        const masthead: HTMLElement | null = doc?.querySelector('dds-cloud-masthead-container');
         const mastheadTopOffset = masthead?.offsetTop;
         this.style.top = `${mastheadTopOffset}px`;
```

Comment:
Just curious, should this line go off only if the const `masthead` above exists?

Reply:
yeah I checked to see in the regular masthead - if `mastheadTopOffset` is null, the `top` inline styling doesn't get applied

Reply:
I could add a conditional in there so it doesn't try to set at all I suppose